### ISSUE TITLE
Update TransparentWebViewPatch

### ISSUE THREAD

**Issue by micheljung** [riccardobl/JavaFXWebEngineDebugger#2]
Hi,

Do you mind that I contact you via this repo? :)
https://gist.github.com/riccardobl/18603f9de508b1ab6c9e stopped working with Java 10.

```
javassist.CannotCompileException: [source error] no such class: com.sun.webkit.graphics.WCRectangle
```

I played around and tried to fix it, but was unable to do so as I don't understand javassist. I was hoping that you do?

**Comment by riccardobl** [comment]
Hi, i'm sorry, but gists still doesn't send notifications :confused:

I don't use java 10, so i haven't ported it yet.  com.sun.webkit.graphics.WCRectangle seems to be still there in java 10, does your code work without the patch? Because you might need to import some modules if it doesn't.

**Comment by micheljung** [comment]
Yes, it's still there and the code to be patched still looks the same. It works without the patch, even with the patch (patching just fails and I get a white WebView background).

My guess is that it has something to do with Java 9 modules but I really don't know. I guess I need to figure out how javassist looks for classes, I was hoping you know :)
I think I'll create a minimal example and ask for help on StackOverflow.
